modules/medical_expense_reports/medical_expense_reports.gemspec

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push File.expand_path('lib', __dir__)
+
+# Maintain your gem's version:
+require 'medical_expense_reports/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |spec|
+  spec.name        = 'medical_expense_reports'
+  spec.version     = MedicalExpenseReports::VERSION
+  spec.authors     = ['Jerek Shoemaker']
```

Comment:
In our other modules we use a team name `['Benefits Pensions']` it might be better to do this for the future.